backend/app/ai_config.py

```
@@ -160,6 +187,234 @@ async def update_ai_config_value(
+
+    if not deleted:
+        raise HTTPException(status_code=404, detail=f"No override found for key '{key}' and user type {user_type_id}")
+
+    return {"success": True, "message": f"Override for '{key}' reverted to global default"}
+
+
+@router.post("/user-type/{user_type_id}/prompts/preview", response_model=PromptPreviewResponse)
+async def preview_prompt_for_user_type(
+    user_type_id: int,
+    request: PromptPreviewRequest,
+    admin: dict = Depends(auth.require_admin)
```

Ruff (0.14.14)

[warning] Unused function argument: `admin`

(ARG001)